README.rst

```diff
@@ -45,7 +45,9 @@ Here is an example configuration file that uses a local SQLite database and the
       HOSTNAME = "http://localhost:5000/minid"
       PORT = 5000
       LANDING_PAGE = "http://localhost:5000/minid/landingpage"
-      GLOBUS_AUTH_ENABLED = False
+
+      GLOBUS_CLIENT_ID = "a5cdede9-567f-4ae5-aba5-cb997d08693a"
+      GLOBUS_CLIENT_SECRET = ""
```

Comment:
The `GLOBUS_AUTH_ENABLED` config option was removed. Not including a Client ID or Secret does effectively the same thing.